src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -2655,89 +2407,73 @@ namespace WaterToAirHeatPumpSimple {
             for (auto &num : HeatReclaim.WaterHeatingDesuperheaterReclaimedHeat)
                 HeatReclaim.WaterHeatingDesuperheaterReclaimedHeatTotal += num;
         }
-        state.dataWaterToAirHeatPumpSimple->QSource -= HeatReclaim.WaterHeatingDesuperheaterReclaimedHeatTotal;
+        simpleWatertoAirHP.QSource -= HeatReclaim.WaterHeatingDesuperheaterReclaimedHeatTotal;
 
         // Update heat pump data structure
-        state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).Power = state.dataWaterToAirHeatPumpSimple->Winput;
-        state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).QLoadTotal = state.dataWaterToAirHeatPumpSimple->QLoadTotal;
-        state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).QSensible = state.dataWaterToAirHeatPumpSimple->QSensible;
-        state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).QLatent =
-            state.dataWaterToAirHeatPumpSimple->QLoadTotal - state.dataWaterToAirHeatPumpSimple->QSensible;
-        state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).QSource = state.dataWaterToAirHeatPumpSimple->QSource;
-        state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).Energy = state.dataWaterToAirHeatPumpSimple->Winput * ReportingConstant;
-        state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).EnergyLoadTotal =
-            state.dataWaterToAirHeatPumpSimple->QLoadTotal * ReportingConstant;
-        state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).EnergySensible =
-            state.dataWaterToAirHeatPumpSimple->QSensible * ReportingConstant;
-        state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).EnergyLatent =
-            (state.dataWaterToAirHeatPumpSimple->QLoadTotal - state.dataWaterToAirHeatPumpSimple->QSensible) * ReportingConstant;
-        state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).EnergySource = state.dataWaterToAirHeatPumpSimple->QSource * ReportingConstant;
+        simpleWatertoAirHP.Power = state.dataWaterToAirHeatPumpSimple->Winput;
+        simpleWatertoAirHP.QLoadTotal = simpleWatertoAirHP.QLoadTotalReport;
+        simpleWatertoAirHP.QSensible = simpleWatertoAirHP.QSensible;
```

Comment:
This line (QSensible=QSensible) is no longer needed, I missed that on the clean up pass. The line above it (QLoadTotal = QLoadTotalReport) was retained since Update uses QLoadTotal to set the energy report.

Reply:
@yzhou601 could you remove lines 2414 and 2415 here and re-run your defect file to make sure it still is fixed?  If so, go ahead and push that final change and I think this can be ready to merge.  Thanks!